src/containers/Chat/ChatMessages/ChatMessages.tsx

```diff
@@ -525,23 +583,34 @@ export const ChatMessages = ({ contactId, collectionId }: ChatMessagesProps) =>
     return false;
   };
 
-  if (conversationInfo && conversationInfo.messages && conversationInfo.messages.length > 0) {
+  if (conversationInfo && conversationInfo.messages && conversationInfo.messages?.length > 0) {
     let reverseConversation = [...conversationInfo.messages];
-    reverseConversation = reverseConversation.map((message: any, index: number) => (
-      <ChatMessage
-        {...message}
-        contactId={contactId}
-        key={message.id}
-        popup={message.id === showDropdown}
-        onClick={() => showEditDialog(message.id)}
-        focus={index === 0}
-        jumpToMessage={jumpToMessage}
-        daySeparator={showDaySeparator(
-          reverseConversation[index].insertedAt,
-          reverseConversation[index + 1] ? reverseConversation[index + 1].insertedAt : null
-        )}
-      />
-    ));
+
+    reverseConversation = reverseConversation.map((message: any, index: number) => {
+      if (groups) {
+        message = {
+          ...message,
+          interactiveContent: '{}',
```

Comment:
Why do we need to add this? I don't think interactive content is relevant in groups context